server/routes/post.route.js

```diff
@@ -18,65 +18,85 @@ router.route('/')
   .get(
     expressJwt({ secret: config.jwtSecret, credentialsRequired: false })
     , loadFullUser
-    , postCtrl.list);
+    , postCtrl.list
+  );
 
 router.route('/recommendations')
-  .get(expressJwt({ secret: config.jwtSecret })
-  , loadFullUser
-  , postCtrl.recommendations);
+  .get(
+    expressJwt({ secret: config.jwtSecret })
+    , loadFullUser
+    , postCtrl.recommendations
+  );
 
 router.route('/:postId')
   .get(
     expressJwt({ secret: config.jwtSecret, credentialsRequired: false })
     , loadFullUser
-    , postCtrl.get);
+    , postCtrl.get
+  );
 
 router.route('/:postId/comments')
-  .get(expressJwt({ secret: config.jwtSecret
-    , credentialsRequired: false }), commentCtrl.list);
+  .get(expressJwt({
+    secret: config.jwtSecret,
+    credentialsRequired: false
+  }), commentCtrl.list);
 
 // Create a comment:
 router.route('/:postId/comment')
   .post(
     expressJwt({ secret: config.jwtSecret })
     , validate(paramValidation.comment)
-    , commentCtrl.create);
+    , commentCtrl.create
+  );
 
 // Get related links associated with postId
 router.route('/:postId/related-links')
-  .get(expressJwt({ secret: config.jwtSecret, credentialsRequired: false })
-    , relatedLinkCtrl.list);
+  .get(
+    expressJwt({ secret: config.jwtSecret, credentialsRequired: false })
+    , relatedLinkCtrl.list
+  );
 
 // Add a related-link to postId:
 router.route('/:postId/related-link')
   .post(
     expressJwt({ secret: config.jwtSecret })
     , validate(paramValidation.relatedLinkCreate)
-    , relatedLinkCtrl.create);
+    , relatedLinkCtrl.create
+  );
 
 router.route('/:postId/upvote')
-  .post(expressJwt({ secret: config.jwtSecret })
-  , transferField({source: 'post', target: 'entity'})
-  , voteCtrl.findVote
-  , voteCtrl.upvote
-  , postCtrl.upvote
-  , voteCtrl.finish
-);
+  .post(
+    expressJwt({ secret: config.jwtSecret })
+    , transferField({ source: 'post', target: 'entity' })
+    , voteCtrl.findVote
+    , voteCtrl.upvote
+    , postCtrl.upvote
+    , voteCtrl.finish
+  );
 
 router.route('/:postId/downvote')
-  .post(expressJwt({ secret: config.jwtSecret })
-  , transferField({source: 'post', target: 'entity'})
-  , voteCtrl.findVote
-  , voteCtrl.downvote
-  , postCtrl.downvote
-  , voteCtrl.finish
-);
+  .post(
```

Comment:
We do need to look into the linter again so we can have more consistency :/ and hopefully prevent random weird merge conflicts.